app/utils/database.py

```diff
@@ -235,6 +235,14 @@ class User(BaseModel):
     validated = BooleanField(default=False)
     privileges = TextField(null=True)
 
+    def frontend_data(self):
+        """
+        Returns the information that is needed in the frontend.
+        """
+        return {'email': self.email,
+                'validated': self.validated if self.validated else False,
```

Comment:
What is `validated`? It does not seem to be able to get this value to be True from the python code.

Reply:
yeah, there used to be both "active" and "validated", and I confused them when I removed "active". I'll sort it out!